README.md

```diff
@@ -102,14 +104,29 @@ cd sourcecred
 yarn install
 yarn backend
 node bin/sourcecred.js load REPO_OWNER REPO_NAME --github-token=GH_TOKEN
+# this loads sourcecred data for a particular repository
 yarn start
 # then navigate to localhost:3000 in your browser
 ```
 
-### Getting Involved
+For example, if you wanted to look at cred for [ipfs/js-ipfs], you could run:
+```
+$ node bin/sourcecred.js load ipfs js-ipfs --github-token=YOUR_GH_TOKEN
+```
+
+[ipfs/js-ipfs]: https://github.com/ipfs/js-ipfs
+
+### Contributing
 
-If you'd like to help out, or you just think SourceCred seems cool and you want
-to learn more, please drop by our [our Discord chat].
+If you would like to contribute to SourceCred:
+* Join our [Discord] and let us know what issue you'd like to work on. We can work
```

Comment:
s/We can work with you to troubleshoot/We can guide you through the architecture/